[CI] LogsIndexingIT testShrink failing

**Build Scans:**
- [elasticsearch-periodic #10386 / release-tests](https://gradle-enterprise.elastic.co/s/lvg4axkou7pzg)
- [elasticsearch-pull-request #89226 / checkpart2_release-tests](https://gradle-enterprise.elastic.co/s/ax4bdw5qlrofg)
- [elasticsearch-pull-request #89180 / checkpart2_release-tests](https://gradle-enterprise.elastic.co/s/q3izxxswxlnsa)
- [elasticsearch-periodic #10380 / release-tests](https://gradle-enterprise.elastic.co/s/qvinwqvvzghfq)
- [elasticsearch-periodic #10374 / release-tests](https://gradle-enterprise.elastic.co/s/cahqvzbvw5sfy)
- [elasticsearch-periodic-java-ea #8 / release-tests](https://gradle-enterprise.elastic.co/s/lmq7bbiqvb5zk)
- [elasticsearch-periodic #10362 / release-tests](https://gradle-enterprise.elastic.co/s/usod4gmsiih3a)
- [elasticsearch-periodic #10356 / release-tests](https://gradle-enterprise.elastic.co/s/kumcldznvdtuk)
- [elasticsearch-periodic #10350 / release-tests](https://gradle-enterprise.elastic.co/s/srz7perzoljcu)
- [elasticsearch-periodic-java-ea #7 / release-tests](https://gradle-enterprise.elastic.co/s/mdnmx5y55ftzw)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:logsdb:internalClusterTest" --tests "org.elasticsearch.xpack.logsdb.LogsIndexingIT.testShrink" -Dtests.seed=5658630F3615790B -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dlicense.key=x-pack/license-tools/src/test/resources/public.key -Dtests.locale=fr-DZ -Dtests.timezone=Etc/GMT+7 -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.logsdb.LogsIndexingIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testShrink'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: [1] shard failures
```

**Issue Reasons:**
- [main] 2 consecutive failures in test testShrink
- [main] 9 consecutive failures in step release-tests
- [main] 2 consecutive failures in step checkpart2_release-tests
- [main] 11 failures in test testShrink (4.0% fail rate in 272 executions)
- [main] 9 failures in step release-tests (100.0% fail rate in 9 executions)
- [main] 2 failures in step checkpart2_release-tests (100.0% fail rate in 2 executions)
- [main] 7 failures in pipeline elasticsearch-periodic (87.5% fail rate in 8 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (3.1% fail rate in 65 executions)
- [main] 2 failures in pipeline elasticsearch-periodic-java-ea (66.7% fail rate in 3 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -507,9 +507,6 @@ tests:
 - class: org.elasticsearch.compute.data.BasicBlockTests
   method: testIntBlock
   issue: https://github.com/elastic/elasticsearch/issues/133596
-- class: org.elasticsearch.xpack.logsdb.patternedtext.PatternedTextFieldMapperTests
-  method: testSyntheticSourceMany
-  issue: https://github.com/elastic/elasticsearch/issues/133598
 - class: org.elasticsearch.compute.data.BasicBlockTests
   method: testDoubleBlock
   issue: https://github.com/elastic/elasticsearch/issues/133606

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -543,9 +543,6 @@ tests:
 - class: org.elasticsearch.xpack.ml.integration.InferenceIT
   method: testInferClassificationModel
   issue: https://github.com/elastic/elasticsearch/issues/133448
-- class: org.elasticsearch.xpack.logsdb.LogsIndexingIT
-  method: testShrink
-  issue: https://github.com/elastic/elasticsearch/issues/133875
 
 # Examples:
 #

